packages/wallets/wallet-turnkey/src/strategy/turnkey/oauth.ts

```
@@ -0,0 +1,93 @@
+    })
+
+    if (result?.credentialBundle) {
+      args.setMetadata({
+        turnkey: {
+          credentialBundle: result.credentialBundle,
+        },
+      })
+      iframeClient.injectCredentialBundle(result.credentialBundle)
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)